env.cpu.yml

```diff
@@ -9,3 +9,5 @@ dependencies:
     - torch-scatter
     - torch-sparse
     - torch-spline-conv
+    - experimental-openvino-python==2021.2.3
```

Comment:
How unstable is openvino right now? I'm a little concerned about adding an experimental package right now (our maintenance overhead is a little heavy already).

I'd be OK with this if you're able to commit to maintaining this feature :)

Reply:
Hi! Actually, it's just about the package. OpenVINO is stable and officially released under `https://pypi.org/project/openvino/`. However it's not compatible with `manylinux` policy and requires extra setup steps (PATH environment variable). So I created a [repo](https://github.com/dkurt/openvino-python-manylinux/) which builds the wheels for the same OpenVINO version but using `manylinux` toolchain and wider Python versions matrix.

Reply:
In the latest commit, switched to official OpenVINO pip distribution (tests passed).
There is still one source package which is installed from GitHub repo: https://github.com/dkurt/openvino/tree/mo_pip/model-optimizer (PR https://github.com/openvinotoolkit/openvino/pull/3123)